### PR TITLE
Update maintainer doc, split out vendor and style doc

### PR DESCRIPTION
**Pull request by belluzj**
This is a rebase of the previous #120 that was never merged + some updates I did while working on the Tamil.

It also includes a script update from @madig, I'll let him comment on the usefulness of that when he's back on Tuesday.

### CODE REVIEW

**Review by madig — APPROVED** [review]

File: docs/MAINTAINER.md [hunk 1]
```diff
@@ -207,21 +145,33 @@ glyphsLib is used to generate Glyphs.app files for those who need it.
...
+
+```bash
+$ python scripts/gs-glyphs2ufo.py source/GoogleSans/staging/*.glyphs
 ```
+
+Assuming the vendor delivered 2 Glyphs.app source files, uprights and italics, the `gs-glyphs2ufo.py` script will convert both to Designspace + UFOs and place them into the same `source/GoogleSans/staging/` folder.
+
+### Steps in common for Designspace + UFO or Glyphs.app imports
+
+3. Import the resulting Designspaces into their intended target Designspaces:
```
Comment (chrissimpkins) [inline docs/MAINTAINER.md:172]:
```suggestion
3. Import the resulting Designspaces into their intended target Designspaces without GRAD axis support:
```

File: docs/MAINTAINER.md [hunk 2]
```diff
@@ -235,15 +185,19 @@ $ python3 scripts/gs-merge-designspace.py \
 
 (Note: if you also have a group list, specify it as an additional switch like so: `--import-groups-file import_groups_italic.txt`)
 
-Now extract the features from the staging UFOs and manually massage them into the existing sources. The font info may also need to be updated, chiefly Unicode and codepage ranges. Additionally, consider which of the imported glyphs need anchor propagation (the list is kept in `scripts/internal/normalize.py`, re-run `scripts/gs-normalize-designspace.py` when you modify it).
+4. Check that glyphs, especially ligatures, have the correct production name set. Something like `k_ssa_uMatra-tamil` should have the production name `uni0B950BCD0BB70BC1`. Using Glyphs.app's naming conventions, you can cross-check names in Glyphs.app's macro panel with the following snippet:
```
Comment (chrissimpkins) [inline docs/MAINTAINER.md:188]: Does this work if you open UFO in Glyphs app as well as a glyphs formatted source file?

**Review by chrissimpkins — COMMENTED** [review]
LGTM. A couple of questions.  I also suggest that we document the glyph deletion workflow and develop a new approach to import over existing glyphs with GRAD axis expansion.

File: docs/MAINTAINER.md (hunk 2 above)
Comment (madig) [reply docs/MAINTAINER.md:188]: You don't need to open the font necessarily, it's a static function. It's enough to use Glyphs.app's naming convention.

File: docs/MAINTAINER.md (hunk 1 above)
Comment (madig) [reply docs/MAINTAINER.md:172]: The import script _does_ import GRAD axes: https://github.com/googlefonts/googlesans/blob/main/scripts/gs-merge-designspace.py#L146-L154. If no GRAD exists, this happens: https://github.com/googlefonts/googlesans/blob/main/scripts/gs-merge-designspace.py#L288-L314

Comment (chrissimpkins) [reply docs/MAINTAINER.md:172]: I misunderstood.  Ty!

File: docs/VENDOR.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+We ask the following from vendors who contribute to the Google Sans project:
+
+* Follow type design conventions
+* Use a required directory and file path structure
+* Name glyphs, glyph classes, kerning groups, and lookups with the required format
+* Develop and submit test documentation that allows us to perform initial and regression testing of shaping, new feature code, and feature code changes that are included in your updates
+
+## Contributing to the Google Sans Project
+
+### Generating Glpyhs.app Source Files
```
Comment (chrissimpkins) [inline docs/VENDOR.md]:
```suggestion
### Generating Glyphs.app Source Files
```

File: docs/VENDOR.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+If you work in Glyphs.app and want to generate sources for modification, skip using Glyphs.app's built-in im- and exporter functionality. Instead, run:
```
Comment (chrissimpkins) [inline docs/VENDOR.md]:
```suggestion
If you work in Glyphs.app and want to generate sources for modification, skip using Glyphs.app's built-in importer / exporter functionality. Instead, run:
```

**Review by chrissimpkins — CHANGES_REQUESTED** [review]

**Review by chrissimpkins — APPROVED** [review]

**Comment by chrissimpkins** [comment]
@googlebot I consent.

**Comment by madig** [comment]
Every time you delete a googlebot comment it posts two new ones.

**Comment by madig** [comment]
@googlebot I consent.

**Comment by chrissimpkins** [comment]
> Every time you delete a googlebot comment it posts two new ones.

Useful little bot...

I thought it was coming from my commits but it appears not

**Comment by chrissimpkins** [comment]
Thank you very much for cleaning this up Jany and Nikolaus!  Looks great.  Merging.

**Pull request merged**